open-apparel-registry/open-apparel-registry
# Add People's Republic of China as alternate country name

Fixes parse errors when contributing lists that use this name.

## Overview

Brief description of what this PR does, and why it is needed.

Connects #868

## Testing Instructions

This assumes the database has been reset with `./scripts/resetdb`

* Check out the `develop` branch.
* Log in as `c2@example.com`
* Browse http://localhost:6543/contribute and submit [prc-test.csv.zip](https://github.com/open-apparel-registry/open-apparel-registry/files/3726337/prc-test.csv.zip)
* Attempt to parse this file and verify that there are 2 failures
```
vagrant@vagrant:/vagrant$ ./scripts/manage batch_process --list-id 16 --action parse
Starting vagrant_database_1 ... done
parse: 1 successes
parse: 2 failures
```
* Check out this branch 
* Browse http://localhost:6543/contribute and submit [prc-test.csv.zip](https://github.com/open-apparel-registry/open-apparel-registry/files/3726337/prc-test.csv.zip)
* Attempt to parse the file and verify that there are 3 successes
```
vagrant@vagrant:/vagrant$ ./scripts/manage batch_process --list-id 17 --action parse
Starting vagrant_database_1 ... done
parse: 3 successes

```
## Checklist
